lib/fluent/plugin/in_tail.rb

```diff
@@ -910,5 +996,26 @@ def read_inode
         @inode
       end
     end
+
+    class PathInodeTuple
```

Comment:
I didn't check `eql?` and other method behaviour of `struct` object but can't use `struct` instead of class for this case?

Reply:
I've checked `eql?` of `struct` - it work as expected, will change to `Struct`